reference/content-types/smart_content.rst

```diff
@@ -4,12 +4,18 @@ Smart content
 Description
 -----------
 
-Shows a list of pages, which depend on a configurable filter. You can define
-which site's children should be included, what tags the filtered pages
-must have, how they are sorted, and how many results you want to get.
-Additionally you can define some presentation types, so that the content 
-manager can decide if the pages should be displayed e.g. in one column or two
-columns. The filter is saved as a JSON string in the database.
+Shows a list of items, which depend on a configurable filter. Depending on
+the DataProvider you can define where the items come from (Datasource),
+what tags the filtered items must have, how they are sorted, and how many
+results you want to get. Additionally you can define some presentation
+types, so that the content manager can decide if the items should be displayed
+e.g. in one column or two columns. The filter is saved as a JSON string in the
+database.
+
+The DataProvider are backend modules which handles the selected filtered values
+and returns the items which fits to this filters. There are some predefined
+ones but you can add your own DataProvider easily. How you can do this is
+described :doc:`here ... </cookbook/smart-content-data-provider>`
```

Comment:
I would not write `here`, but something like `this is described in :doc:smart-content-data-provider` or however the path is. Then the correct link title is shown, and it is more clear to the user where the link leads to.